tests/unit/test_mbpt.cpp

```diff
@@ -326,20 +326,20 @@ TEST_CASE("NBodyOp", "[mbpt]") {
             L"{{{\\frac{1}{2}}}{A^{{i_1}}_{{a_1}{a_2}}}{\\tilde{a}^{{a_1}{a_2}}"
             L"_{\\textvisiblespace\\,{i_1}}}}");
 
-    auto P_0_1 = P(0, 1)->as<op_t>();
+    auto P_0_1 = P(nₚ(1), nₕ(0))->as<op_t>();
     //    std::wcout << "P_0_1: " << to_latex(simplify(P_1_0.tensor_form()))
     //               << std::endl;
     REQUIRE(to_latex(simplify(P_0_1.tensor_form())) ==
             L"{{A^{}_{{i_1}}}{\\tilde{a}^{{i_1}}}}");
```

Comment:
this clearly does not match `P_0_1` specification, since `nₕ` is 0

Reply:
The same for the next two tests, `P_2_1` and `P_2_3` are also not correct

Reply:
fixed in https://github.com/ValeevGroup/SeQuant/pull/228/commits/4d9fa6c118c9e362a8d6256c1d48f7423d70eeff ... this is probably why EOM-{IP,EA}-CC did not work

Reply:
I think I manually checked everything when I initially wrote the tests, but it probably got mixed up later. I will be more careful next time. On that note, EOM-CCSD-R tests have been verified numerically, but not EOM-CCSD-L. We can remove it if not needed.